observability/probes/memory.ts

```diff
@@ -0,0 +1,18 @@
+import { getProbeThresholdAsNum, type LiveChecker } from "./handler.ts";
+
+const HEALTHY_RATIO = Deno.env.get("PROBE_MEMORY_HEALTHY_RATIO");
+
+const healthyRatio = HEALTHY_RATIO ? parseFloat(HEALTHY_RATIO) : 0.81;
```

Comment:
Why should we care about HEALTHY_RATIO?

It seems a clumsy or unnecessary metric. RSS max total should what we care.

Reply:
But, if the memory reaches the RSS so the pod will get killed by k8s anyways, how it should be then?